### ISSUE TITLE
Water problem

### ISSUE THREAD

**Issue by JasperTheMinecraftDev** [PaperSpigot/Paper#8016]
### Expected behavior

I expected to see normal water LMAO

### Observed/Actual behavior

Uhm, something very weird, this is the image: https://imgur.com/a/dUEXW7g

Also, if I see water, I see many white space between 2 water sources, here comes an image: https://imgur.com/a/OoXZ5QR

Oh bruh, the bugs keep coming ;))
let's go, here comes number #3
https://imgur.com/a/p3m4mjl , you gonna look very carefully, that line from the 2 water types is so weird...

So in short, all water is bugged and it looks very ugly bruh
Hope ya can fix it!

### Steps/models to reproduce

So, get some ice and place water on it,
Then go under the ice, and you'll see the most breaked game ever ;))

### Plugin and Datapack List

No plugins

### Paper version

1.19, build 8

### Other

I wish you all a good day :)

**Comment by kennytv** [comment]
That's not a Paper or server issue

**Comment by JasperTheMinecraftDev** [comment]
Uhm, what's it then?

**Comment by JasperTheMinecraftDev** [comment]
Everyone on my server get's it..

**Comment by MiniDigger** [comment]
a) you are severely outdated and b) this is a client issue, try without mods and the vanilla launcher, if it doesnt go away report to mojang

**Comment by JasperTheMinecraftDev** [comment]
I use iris + sodium, the others use vanilla
